sklearn/utils/__init__.py

```diff
@@ -590,3 +591,84 @@ def is_scalar_nan(x):
     # Redondant np.floating is needed because numbers can't match np.float32
     # in python 2.
     return bool(isinstance(x, (numbers.Real, np.floating)) and np.isnan(x))
+
```

Comment:
PEP8 says that you should have 2 line returns here

Reply:
Addressed in https://github.com/lebigot/scikit-learn/commit/6bc2a39fe5e6a9f65c9d54fb442e18a265e82e9a